src/components/DataRoom/Hero/AsciiWrapper.tsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useMemo } from 'react'
+import { ASCIIEffect } from './utils'
+
+export default function AsciiWrapper() {
```

Comment:
```suggestion
const AsciiWrapper = () =>  {
```
and add the return type